ntoskrnl/config/i386/cmhardwr.c

```diff
@@ -241,6 +247,9 @@ CmpInitializeMachineDependentConfiguration(IN PLOADER_PARAMETER_BLOCK LoaderBloc
     CPU_INFO CpuInfo;
     ULONG ExtendedId;
     PKPRCB Prcb;
+#ifdef _M_AMD64
+    PCHAR CmpID;
```

Comment:
I don't see the point - why complicate the code and detect CPU vendor for that string? Where is it going to be used?
They share the same instruction set

Reply:
what string?

Reply:
This string affects ACPI identifiers, and they affect what CPU driver will be assigned to the device. We don't have specialized CPU drivers though, but this is to match Windows behavior.

Reply:
So we are going to have one driver for all CPUs on 32bit and 3 drivers on 64bit? Where is the logic here?

Reply:
We will have one driver for both cases, until that changes for some reason. Note that Windows also have different CPU drivers on 32-bit.